Mathlib/MeasureTheory/Integral/SetIntegral.lean

```diff
@@ -1379,6 +1379,29 @@ variable {X Y F G 𝕜 : Type*} [TopologicalSpace X]
 
 open Metric ContinuousLinearMap
 
+/-- The parametric integral over a continuous function on a compact set is continuous,
+  under mild assumptions on the topologies involved. -/
+theorem continuous_parametric_integral_of_continuous
+    [FirstCountableTopology X] [LocallyCompactSpace X]
+    [OpensMeasurableSpace Y] [SecondCountableTopologyEither Y E] [IsLocallyFiniteMeasure μ]
+    {F : X → Y → E} (hF : Continuous fun p : X × Y ↦ F p.1 p.2) {s : Set Y} (hs : IsCompact s) :
+    Continuous fun x ↦ ∫ a in s, F x a ∂μ := by
+  rw [continuous_iff_continuousAt]
+  intro x₀
+  rcases exists_compact_mem_nhds x₀ with ⟨U, U_cpct, U_nhds⟩
+  rcases (U_cpct.prod hs).bddAbove_image hF.norm.continuousOn with ⟨M, hM⟩
+  apply continuousAt_of_dominated
+  · exact eventually_of_forall fun x ↦ (hF.comp (Continuous.Prod.mk x)).aestronglyMeasurable
+  · refine Eventually.mono U_nhds fun x x_in ↦ ?_
+    rw [ae_restrict_iff]
+    · exact eventually_of_forall fun t t_in ↦ hM (mem_image_of_mem _ <| mk_mem_prod x_in t_in)
+    · exact (isClosed_le (hF.comp <| Continuous.Prod.mk x).norm continuous_const).measurableSet
+  · exact integrableOn_const.mpr (Or.inr hs.measure_lt_top)
+  · apply ae_of_all
+    intro a
+    -- TODO: can fun_prop show this?
```

Comment:
Well, can it? 😄 If not, since `fun_prop` is under active development right now, it might be worth posting this example on Zulip.